Makefile

```diff
@@ -7,24 +7,17 @@ LAST_TAG := "$(shell git rev-list --tags --max-count=1)"
 OPMS_VERSION := "$(shell git describe --tags ${LAST_TAG})-next"
 PROTON_COMMIT := "2a30976f7b40884ddd90e1792576c0941426e8bc"
 
-.PHONY: build test generate pack-files generate-proto unit-test smoke-test integration-test vet coverage clean install lint
+.PHONY: build test generate-proto unit-test smoke-test integration-test vet coverage clean install lint
 
 .DEFAULT_GOAL := build
 
-build: generate # build optimus binary
+build: # build optimus binary
 	@echo " > building optimus version ${OPMS_VERSION}"
 	@go build -ldflags "-X ${NAME}/config.Version=${OPMS_VERSION} -X ${NAME}/config.BuildCommit=${LAST_COMMIT}" -o optimus .
 	@echo " - build complete"
 	
 test: smoke-test unit-test vet ## run tests
 
-generate: pack-files
-	@echo " > notice: skipped proto generation, use 'generate-proto' make command"
-	
-pack-files:
```

Comment:
We need to also update the github actions to remove the invocation of pack-files.
https://github.com/odpf/optimus/blob/main/.github/workflows/lint.yml#L21

Reply:
done